pt-BR/mail/chrome/messenger/focusPane.dtd

```diff
@@ -1,73 +1,73 @@
 <!-- strings used in both the focus pane and the full text search focus pane -->
-<!ENTITY focusPaneFilterByTags.label    "Favorite Topics">
-<!ENTITY focusPaneFilterByTagsMac.label "FAVORITE TOPICS">
-<!ENTITY focusPaneAllTags.label         "All">
-<!ENTITY focusPaneNoTags.label          "None">
+<!ENTITY focusPaneFilterByTags.label    "Tópicos favoritos">
+<!ENTITY focusPaneFilterByTagsMac.label "TÓPICOS FAVORITOS">
+<!ENTITY focusPaneAllTags.label         "Todos">
+<!ENTITY focusPaneNoTags.label          "Nenhum">
 
 <!-- strings used only in the full text search focus pane -->
-<!ENTITY focusPaneSearchScopeTitle.label       "Search">
-<!ENTITY focusPaneSearchScopeTitleMac.label    "SEARCH">
-<!ENTITY focusPaneSearchScopeEverywhere.label  "All Accounts">
-<!ENTITY focusPaneSearchAttribute.label        "Attributes">
-<!ENTITY focusPaneSearchAttributeMac.label     "ATTRIBUTES">
-<!ENTITY focusPaneHasAttachmentAttribute.label           "Has Attachment">
-<!ENTITY focusPaneUnreadAttribute.label           "Unread">
-<!ENTITY focusPaneFilterByDate.label    "Date">
-<!ENTITY focusPaneFilterByDateMac.label "DATE">
-<!ENTITY focusPaneAnyDate.label         "Any Date">
-<!ENTITY focusPaneToday.label           "Today">
-<!ENTITY focusPaneYesterday.label       "Yesterday">
-<!ENTITY focusPaneThisWeek.label        "This Week">
-<!ENTITY focusPaneThisMonth.label       "This Month">
-<!ENTITY focusPaneFilterByContact.label    "Favorite Contacts">
-<!ENTITY focusPaneFilterByContactMac.label "FAVORITE CONTACTS">
+<!ENTITY focusPaneSearchScopeTitle.label       "Pesquisar">
+<!ENTITY focusPaneSearchScopeTitleMac.label    "PESQUISAR">
+<!ENTITY focusPaneSearchScopeEverywhere.label  "Todas as contas">
+<!ENTITY focusPaneSearchAttribute.label        "Atributos">
+<!ENTITY focusPaneSearchAttributeMac.label     "ATRIBUTOS">
+<!ENTITY focusPaneHasAttachmentAttribute.label           "Tem anexo">
+<!ENTITY focusPaneUnreadAttribute.label           "Não-lido">
+<!ENTITY focusPaneFilterByDate.label    "Data">
+<!ENTITY focusPaneFilterByDateMac.label "DATA">
+<!ENTITY focusPaneAnyDate.label         "Qualquer data">
+<!ENTITY focusPaneToday.label           "Hoje">
+<!ENTITY focusPaneYesterday.label       "Ontem">
+<!ENTITY focusPaneThisWeek.label        "Esta semana">
+<!ENTITY focusPaneThisMonth.label       "Este mês">
+<!ENTITY focusPaneFilterByContact.label    "Contatos favoritos">
+<!ENTITY focusPaneFilterByContactMac.label "CONTATOS FAVORITOS">
 
 <!-- focus pane topics context menus -->
-<!ENTITY pbFavoriteTopicsContextMenu-edit.label         "Rename">
+<!ENTITY pbFavoriteTopicsContextMenu-edit.label         "Renomear">
 <!ENTITY pbFavoriteTopicsContextMenu-edit.accesskey     "n">
-<!ENTITY pbFavoriteTopicsContextMenu-favorite.label     "Remove From Favorites">
+<!ENTITY pbFavoriteTopicsContextMenu-favorite.label     "Remover dos favoritos">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY pbFavoriteTopicsContextMenu-favorite.label     "Excluir dos favoritos">